### PR TITLE
hide the thumbnails column behind an experiment flag in the section projects view

### PR DESCRIPTION
**Pull request by davidsbailey**
We have decided to launch Teacher View of Student Projects without the thumbnails column while we continue to work on thumbnails. This PR hides the thumbnails column by default so that it will look like this:
screen shot 2017-04-06 at 7 33 13 pm

To see project thumbnails: https://code.org/teacher-dashboard?sectionProjects=1&enableExperiments=showProjectThumbnails#/sections/

The view with project thumbnails enabled looks like just like it did before:
screen shot 2017-04-06 at 7 32 10 pm

To hide project thumbnails again:
* https://code.org/teacher-dashboard?sectionProjects=1&disableExperiments=showProjectThumbnails#/sections/

### Launch plan
1. merge + ship this PR, then wait a deploy or two
2. run `DCDO.set('sectionProjects', true)` at which point section projects will be live
3. after a deploy or two, remove all of the logic related to the `sectionProjects` and `section_projects` flags.

### CODE REVIEW

File: apps/test/unit/templates/projects/SectionProjectsListTest.js [hunk 1]
```diff
@@ -65,6 +65,24 @@ function assertRowThumbnail(rowElement, thumbnailUrl) {
...
+ * @param {string} appType Expected app type (App Lab, Game Lab, etc)
+ * @param {string} lastEdited Expected last edited date (Month DD, YYYY). Note that this
+ *   format is used only in unit tests due to incorrect date formatting in PhantomJS.
+ *   The desired date format which we will show in all browsers is MM/DD/YYYY.
+ */
+function assertRowContentsWithoutThumbnail(rowElement, projectName, studentName, appType, lastEdited) {
+  expect(rowElement.childNodes[COLUMNS_WITHOUT_THUMBNAILS.PROJECT_NAME].innerText).to.equal(projectName);
+  expect(rowElement.childNodes[COLUMNS_WITHOUT_THUMBNAILS.STUDENT_NAME].innerText).to.equal(studentName);
+  expect(rowElement.childNodes[COLUMNS_WITHOUT_THUMBNAILS.APP_TYPE].innerText).to.equal(appType);
+  // Temporarily comment out this line to make tests pass locally in Chrome
```
Comment (Bjvanminnen) [inline apps/test/unit/templates/projects/SectionProjectsListTest.js:81]: why is this necessary?

Comment (davidsbailey) [reply apps/test/unit/templates/projects/SectionProjectsListTest.js:81]: In PhantomJS, `Date.toLocaleDateString()` has a non-standard output format of `December 29, 2016` instead of the standard `12/29/2016` output by Chrome. The Date class has options to explicitly specify the output format:
```
var options = {year: 'numeric', month: 'numeric', day: 'numeric' };
console.log(date.toLocaleDateString('en-US', options));
```
However PhantomJS doesn't respect these options either.
